erpnext/crm/doctype/opportunity/opportunity.js

```diff
@@ -87,6 +88,9 @@ erpnext.crm.Opportunity = class Opportunity extends frappe.ui.form.Controller {
 
 					me.frm.add_custom_button(__("Vehicle Quotation"), () => me.make_vehicle_quotation(),
 						__('Create'));
+
+					me.frm.add_custom_button(__("Test Drive Gate Pass"), () => me.make_opportunity_gate_pass(),
+						__('Create'));
```

Comment:
Check if vehicles domain is enabled AND if conversion document == "Order"

Reply:
Already checking that